docs/CREDENTIALS.md

```diff
@@ -288,6 +320,13 @@ services where they are ultimately consumed.
    initrds, as userspace can place credentials next to these EFI kernels, and
    be sure they can be accessed securely from initrd context.
 
+5. Credentials may be passed from the initrd to the host during the initrd →
```

Comment:
as earlier I'd swap the cmdline and this one, and say that the cmdline should be a last resort or so